frontend/src/stores/auth.js

```diff
@@ -67,6 +69,16 @@ export const useAuthStore = defineStore('auth', {
           // Lookup the permissions
           this.permissions = appStore.roles.find((role) => role.roleId === this.userInfo.role?.ofm_portal_roleid)?.permissions.map((p) => p.permissionName)
 
+          // A facility can apply for a core application if there is an Open Intake or it is in the allowed facility list of a Limited Intake
+          this.userInfo?.facilities?.forEach((facility) => {
+            facility.isAddCoreApplicationAllowed = appStore.applicationIntakes?.some((intake) => {
+              const allowedFacilities = intake?.facilities?.map((facility) => facility.facilityId)
+              const isWithinApplicationIntakeWindow = moment().isSameOrAfter(moment(intake.startDate)) && moment().isSameOrBefore(moment(intake.endDate))
+              const isOpenIntake = intake.type === APPLICATION_INTAKE_TYPES.OPEN_INTAKE
+              const isFacilityAllowedToApply = allowedFacilities?.includes(facility.facilityId)
+              return (isOpenIntake && isWithinApplicationIntakeWindow) || (isWithinApplicationIntakeWindow && isFacilityAllowedToApply)
```

Comment:
This could be simplified as `isWithinApplicationIntakeWindow && (isOpenIntake || isFacilityAllowedToApply)`.

Reply:
great suggestions! thanks Wes! it is updated